util/screenshot/package.lisp

```diff
@@ -0,0 +1,14 @@
+;;;; package.lisp
+
+(defpackage #:screenshot
+  (:use #:cl :stumpwm :zpng))
+
+(in-package #:screenshot)
+
+(import
```

Comment:
This import is unnecessary, all of those symbols are external to the `ZPNG` package and will thus already be `:use`'d by the `defpackage` above.